### PR TITLE
add version to message contents

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/messaging/issues/287

## This introduces a breaking change

- [x] Yes
- [ ] No

## This should yield a new module release

- [ ] Yes
- [ ] No

This will cause all messages whose topic is determined by version to be lost when investigator updates to the newest version of messaging

## This Pull Request implements

Simply add version to message contents. We don't expect this to be set by the user/producer so it is given a default value representing the current iteration

### CODE REVIEW

File: thoth/messaging/advise_justification.py [hunk 1]
```diff
@@ -29,14 +29,14 @@ class AdviseJustificationMessage(MessageBase):
...
     topic_name = "thoth.advise-reporter.advise-justification"
-    _message_version = 1  # update on schema change
 
     class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
         """Class used to represent contents of advise justification message Kafka topic."""
 
         message: str
         justification_type: str
         count: int
+        version: str = "v1"
```
Comment (pacospace) [inline thoth/messaging/advise_justification.py:39]: How do we guarantee investigator can handle v1 and v2 at the same time using same version of thoth-messaging?

Comment (KPostOffice) [reply thoth/messaging/advise_justification.py:39]: https://github.com/thoth-station/investigator/pull/300

Comment (pacospace) [reply thoth/messaging/advise_justification.py:39]: But if I update thoth-messaging I cannot look back to v1 if we moved to v2 of messages right?

**Review by pacospace — APPROVED** [review]
Nice!! Thanks :100: :1st_place_medal:

**Pull request merged**